src/components/ClusterManager.js

```diff
@@ -363,6 +388,23 @@ export default ajaxCaller(class ClusterManager extends PureComponent {
     this.resetUpdateInterval()
   }
 
+  async componentDidUpdate(prevProps) {
+    const { ajax: { Jupyter } } = this.props
+    const prevCluster = _.last(_.sortBy('createdDate', _.remove({ status: 'Deleting' }, prevProps.clusters))) || {}
+    const { status, jupyterUserScriptUri, googleProject, clusterName, stagingBucket } = this.getCurrentCluster() || {}
+
+    if (status === 'Error' && prevCluster.status !== 'Error' && !errorNotified.current) {
+      const { errors: clusterErrors } = await Jupyter.cluster(googleProject, clusterName).details()
```

Comment:
Don't we already have this info from the cluster list?

Reply:
No, cluster list returns the basic cluster information (everything in the cluster table), whereas cluster details gives you all the cluster info by doing a join between all the tables (images, instances, errors, and scopes)